### PR TITLE
Add building of the VSIX to the build process

### PR DESCRIPTION
**Pull request by sergeybykov**
Let's see if CI will succeed in building this.

### CODE REVIEW

**Comment by sergeybykov** [comment]
If somebody, especially F#-savvy folks, has a clue about what's going on here in CI, I'd appreciate your help.
> "d:\j\workspace\dotnet_orleans\innerloop_prtest\src\OrleansVSTools\VSProjectTemplateGrainImplementationFSharp\VSProjectTemplateGrainImplementationFSharp.fsproj" (default target) (17) ->
         C:\Windows\Microsoft.NET\Framework\v4.0.30319\Microsoft.Common.targets(2098,7): error MSB4057: The target "CreateManifestResourceNames" does not exist in the project. [d:\j\workspace\dotnet_orleans\innerloop_prtest\src\OrleansVSTools\VSProjectTemplateGrainImplementationFSharp\VSProjectTemplateGrainImplementationFSharp.fsproj]

**Comment by sergeybykov** [comment]
I removed F# projects from the solution for now. They are broken anyway. Let's see if it'll make it build in CI.

File: src/Build.cmd [hunk 1]
```diff
@@ -49,14 +49,11 @@ SET OutDir=%CMDHOME%\..\Binaries\%CONFIGURATION%
...
-@REM
-@REM Install Visual Studio SDK and uncomment the following lines 
-@REM to build Visual Studio project templates.
-@REM
-@REM "%MSBUILDEXE%" /nr:False /m /p:Configuration=%CONFIGURATION% "%CMDHOME%\OrleansVSTools\OrleansVSTools.sln"
-@REM xcopy /s /y %CMDHOME%\SDK\VSIX %OutDir%\VSIX\
-@REM @if ERRORLEVEL 1 GOTO :ErrorStop
-@REM @echo BUILD ok for VSIX package for %PROJ%
+
+"%MSBUILDEXE%" /nr:False /m /p:Configuration=%CONFIGURATION% "%CMDHOME%\OrleansVSTools\OrleansVSTools.sln"
```
Comment (jdom) [inline src/Build.cmd]: you should probably skip this step if BuildOrleansNuGet env variable is false

**Comment by sergeybykov** [comment]
It seems to build fine now but tests are failing, which is surprising, considering that this should only add OrleansVSTools.dll to the output folder, and it doesn't even reference Orleans.dll.

@jdom Do you have any thoughts on what's going on here?

**Comment by jdom** [comment]
@dotnet-bot test this please

**Comment by jdom** [comment]
no, not sure, but it is something related to dynamically loading the binaries in the output directory... Did you try building and running the tests locally?
if it's not trivial, we can also build this as a separate command after all the tests were run... after all there is no purpose in building them if it's an unstable build...

**Comment by sergeybykov** [comment]
Tests seem to run fine locally:
>=== TEST EXECUTION SUMMARY ===
   Tester  Total: 140, Errors: 0, Failed: 0, Skipped: 7, Time: 438.603s

**Comment by jdom** [comment]
Some of the project templates are referencing NewtonSoft.Json 6.0.8 so it's overriding in the Output folder the version that Orleans uses.

**Comment by sergeybykov** [comment]
I think this can be merged now. There are still further improvements, but at least it gets built fine now.

**Pull request merged**